module/files/helper.js

```diff
@@ -26,61 +27,141 @@ module.exports = class FilesHelper{
+            try {
 
-            let result;
-            if( process.env.CLOUD_STORAGE === "AWS" ) {
-                result = await awsServices.uploadFile(
-                    file,
-                    filePathForBucket,
-                    bucketName
-                );
-            } else if( process.env.CLOUD_STORAGE === "GC" ) {
-                result = await googleCloudServices.uploadFile(
-                    file,
-                    filePathForBucket,
-                    bucketName
-                )
-            }
+                let result;
+                if (process.env.CLOUD_STORAGE === constants.common.AWS_SERVICE) {
+                    result = await awsServices.uploadFile(
+                        file,
+                        filePathForBucket,
+                        bucketName
+                    );
+                } else if (process.env.CLOUD_STORAGE === constants.common.GOOGLE_CLOUD_SERVICE) {
+                    result = await googleCloudServices.uploadFile(
+                        file,
+                        filePathForBucket,
+                        bucketName
+                    )
+                } else if (process.env.CLOUD_STORAGE === constants.common.AZURE_SERVICE) {
+                    result = await azureService.uploadFile(
+                        file,
+                        filePathForBucket,
+                        bucketName
+                    )
+                }
 
-            return resolve(result);
+                return resolve(result);
             } catch (error) {
                 return reject(error);
             }
         })
-   }
+    }
 
-   /**
-      * Get downloadable url
-      * @method
-      * @name getDownloadableUrl
-      * @param  {filePath}  - File path.
-      * @return {String} - Downloadable url link
-    */
+    /**
+       * Get downloadable url
+       * @method
+       * @name getDownloadableUrl
+       * @param  {filePath}  - File path.
+       * @return {String} - Downloadable url link
+     */
 
-   static getDownloadableUrl( filePath,bucketName ) {
-       return new Promise(async (resolve, reject) => {
-           try {
-               let result;
-               
-               if( process.env.CLOUD_STORAGE === "AWS" ) {
-                   result = await awsServices.getDownloadableUrl(
-                       filePath,
-                       bucketName
-                    );
-                } else if( process.env.CLOUD_STORAGE === "GC" ) {
-                    result = await googleCloudServices.getDownloadableUrl(
+    static getDownloadableUrl(filePath, bucketName, storageName = "") {
+        return new Promise(async (resolve, reject) => {
+            try {
+
+                let cloudStorage = process.env.CLOUD_STORAGE;
+
+                if (storageName !== "") {
+                    cloudStorage = storageName;
+                }
+
+                if (Array.isArray(filePath) === true) {
```

Comment:
Array.isArray(filePath) will return a true statement only. No need to check for === true statement. Shouldn't we check for array length also if empty array will be given then this statement will be true and error will be thrown right ??